security/pkg/nodeagent/sds/server.go

```diff
@@ -193,15 +202,23 @@ func (s *Server) initGatewaySdsService(options *Options) error {
 	}
 
 	go func() {
+		log.Info("Start SDS grpc server for ingress gateway proxy")
+		numErrorOutputTime1 := 0
```

Comment:
Time1 and Time2 are not good name for var.

Reply:
Done: revised based on the comments.